__tests__/api/app.test.js

```diff
@@ -0,0 +1,47 @@
+const request = require('supertest');
+const app = require('../../app');
+const seedTestData = require('../../db/data/test-data');
+const seed = require('../../db/seeds/seed');
+const db = require('../../db/connection');
+
+afterAll(() => {
+  return db.end();
+});
+
+beforeEach(() => {
+  return seed(seedTestData);
+});
+
+describe('Testing app', () => {
+  describe('Endpoint /api/topics', () => {
+    describe('Method GET', () => {
+      test('GET: 200 status', () => {
+        return request(app).get('/api/topics').expect(200);
+      });
+
+      test('GET: 200 status with all topics data and properties', () => {
+        return request(app)
+          .get('/api/topics')
+          .expect(200)
+          .then(({ body }) => {
+            const topics = body;
+            expect(Array.isArray(topics)).toBe(true);
+
```

Comment:
Don't forget to check the length of the array too as it should always be the same. We want to avoid automatically passing tests with an empty array.